src/java/org/apache/cassandra/index/sai/disk/v1/IndexWriterConfig.java

```diff
@@ -57,6 +60,10 @@ public class IndexWriterConfig
 
     public static final int DEFAULT_MAXIMUM_NODE_CONNECTIONS = 16;
     public static final int DEFAULT_CONSTRUCTION_BEAM_WIDTH = 100;
+    // Use non-positive value to disable it. When num of vectors in vector memtable index reaches the threshold, it triggers flush
+    public static final int DEFAULT_VECTOR_FLUSH_TRESHOLD = Integer.getInteger("dse.default_vector_flush_threshold", -1);
+    // Use non-positive value to disable it. Period in ms to schedule a flush for vector memtable index.
+    public static final int DEFAULT_VECTOR_FLUSH_PERIOD_IN_MS = Integer.getInteger("dse.default_vector_flush_period_in_ms", -1);
```

Comment:
Nit: How about `Duration.getDurationMilliseconds(Duration.from(value))`?

Reply:
Moved to `CassandraRelevantProperties`